dev/PullToRefresh/ScrollViewerIRefreshInfoProviderAdapter/InteractionTests/ScrollViewerAdapterTests.cs

```diff
@@ -40,7 +40,8 @@ public void TestCleanup()
             TestCleanupHelper.Cleanup();
         }
 
-        // [TestMethod] Disabled pending investigation of 11754081
+        [TestMethod] //Disabled pending investigation of 11754081
```

Comment:
Just putting this out here as a thought: Maybe we should update this references to point to github issues, and if there isn't a corresponding issue yet create one.

Reply:
I agree, we should get these on github..

Reply:
Would you like me to create those issues and update the refs with a PR? Should every disabled test be it's own issue or should we group them?

Also how would we find out if a test is reliable again? Run the CI 10 times? Some of the tests look really simple, yet were disabled, sometimes with comments that don't really explain what the issue is.

Reply:
Sounds great. Lets do one issue with a checklist, that way we have one place to see that information.